test/requests/hover_expectations_test.rb

```diff
@@ -181,7 +181,6 @@ def test_hovering_over_gemfile_dependency
 
     assert_includes(response.contents.value, spec.name)
     assert_includes(response.contents.value, spec.version.to_s)
-    assert_includes(response.contents.value, spec.description)
```

Comment:
I think CI was failing on this line. Since we're stripping the description, I don't think we should assert on it anyways.